src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/Extensions/FieldBuilderResolverExtensions.cs

```diff
@@ -1,14 +1,14 @@
 using System;
 using System.Threading.Tasks;
+using GraphQL;
 using GraphQL.Builders;
-using GraphQL.Types;
 using OrchardCore.Apis.GraphQL.Resolvers;
 
 namespace OrchardCore.Apis.GraphQL
 {
     public static class FieldBuilderResolverExtensions
     {
-        public static FieldBuilder<TSourceType, TReturnType> ResolveLockedAsync<TSourceType, TReturnType>(this FieldBuilder<TSourceType, TReturnType> builder, Func<ResolveFieldContext<TSourceType>, Task<TReturnType>> resolve)
+        public static FieldBuilder<TSourceType, TReturnType> ResolveLockedAsync<TSourceType, TReturnType>(this FieldBuilder<TSourceType, TReturnType> builder, Func<IResolveFieldContext<TSourceType>, Task<TReturnType>> resolve)
```

Comment:
we need to look if we can remove `ResolveLockedAsync` now dataloaders support serial execution strategies (we switched to a custom parallel strategy that uses locks to allow dataloader support; but a serial strategy with no locks is probably more performant and less error prone (ie, someone forgot to use ResolveLockedAsync instead of ResolveAsync which is the normal graphqldotnetresolver) - could be saved for another PR though imo.